.github/workflows/release.yaml

```diff
@@ -91,13 +92,18 @@ jobs:
           echo RUNTIME VERSION: ${{env.RUNTIME_VERSION}}
           echo GITHUB_EVENT_NAME: ${GITHUB_EVENT_NAME}
       
+      - name: Create Readme.txt file
+        shell: bash
+        run: |
+          cp ${{env.README_MD_FILE}} ${{env.README_TXT_FILE}}
```

Comment:
Have a quick question regarding this, won't the formatting chars that we use for MD like `#`, backtick etc. be as it is in the README text file?

Reply:
The characters would be there. But that should be okay as anyway it's a readme and the formatting characters do improve readabiliity. 
 
Having a separate readme text file in repo itself is something I don't like to do as then any change in Readme.md would need a change in txt file as well, causing redundancy and inconsistency between two

Reply:
Or else we can keep .md file itself in the bundle if characters cause an issue?